HLTriggerOffline/SUSYBSM/python/SUSYBSM_alphaT_cff.py

```diff
@@ -2,111 +2,108 @@
 
 SUSY_HLT_HT200_alphaT0p57 = cms.EDAnalyzer("SUSY_HLT_alphaT",
   trigSummary = cms.InputTag("hltTriggerSummaryAOD",'', 'HLT'), #to use with test sample
-  #trigSummary = cms.InputTag("hltTriggerSummaryAOD"),
-  #pfMETCollection = cms.InputTag("pfMet"),
   caloJetCollection = cms.InputTag("ak4CaloJets"),
-  #caloJetCollection = cms.InputTag("hltAK4CaloJets"),
-  #pfJetCollection = cms.InputTag("ak4PFJetsCHS"),
+  pfJetCollection = cms.InputTag("ak4PFJetsCHS"),
   TriggerResults = cms.InputTag('TriggerResults','','HLT'), #to use with test sample
-  #TriggerResults = cms.InputTag('TriggerResults','','HLT'),
   HLTProcess = cms.string('HLT'),
-  TriggerPath = cms.string('HLT_HT200_DiJet90_AlphaT0p57_v'),
-  TriggerPathAuxiliaryForHadronic = cms.string('HLT_IsoTkMu24_eta2p1_IterTrk02_v'),
-  TriggerFilter = cms.InputTag('hltHT200CaloAlphaT0p57', '', 'HLT'), #the last filter in the path
+  TriggerPath = cms.string('HLT_PFHT200_PFDiJet90_PFAlphaT0p57_v'),
```

Comment:
The name of the path is HLT_PFHT200_DiPFJet90_PFAlphaT0p57 (and the same for the other three similar paths dealth with in this pull request), see JIRA ticket CMSHLT-207
Please provide a fix with the correct HLT path name: only at that point those paths will be allowed in the HLT GRun menu.